docs_src/microservices/general/index.md

```diff
@@ -170,6 +170,9 @@ In EdgeX Foundry, certain characters in metadata (like device names, resource na
 | `&`       | Ampersand               | `%26`              |
 | `?`       | Question Mark / Query   | `%3F`              |
```

Comment:
if we suggest those chararcters should be used when the `NameFieldEscape` config is enabled, remove them from this section

Reply:
Rewrote this table and example.